### PR TITLE
Make beacon chain broadcast shard state

### PR DESCRIPTION
**Pull request by rlan35**
peerID: 9002, numOfConsensus: 30
peerID: 9016, numOfConsensus: 26
peerID: 9017, numOfConsensus: 26
peerID: 9003, numOfConsensus: 30
peerID: 9001, numOfConsensus: 30
peerID: 9014, numOfConsensus: 24
peerID: 9004, numOfConsensus: 30
peerID: 9011, numOfConsensus: 24
peerID: 9007, numOfConsensus: 24
peerID: 9013, numOfConsensus: 24
peerID: 9012, numOfConsensus: 24
peerID: 9006, numOfConsensus: 24
peerID: 9008, numOfConsensus: 24
peerID: 9009, numOfConsensus: 24
peerID: 9019, numOfConsensus: 26
peerID: 9018, numOfConsensus: 26'

### CODE REVIEW

**Comment by rlan35** [comment]
Jenkins, test this please.

**Comment by rlan35** [comment]
Jenkins, test this please.

File: test/chain/.bnkey [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (chaosma) [inline test/chain/.bnkey]: what is this key used for?

Comment (rlan35) [reply test/chain/.bnkey]: deleted. It's a mistake.

**Pull request merged**